plenary/plenary-22.yaml

```diff
@@ -0,0 +1,108 @@
+    identifier: 251
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved
+
+    actions:
+      - type: resolves
+        date_effective: 2004-09-10
+        message: Pending approval of the ballot on 15000-Part 5, ISO/TC154 resolves to request the establishment of a formal liaison between the ISO/TC154 project team for TS 15000-Part 5 and ISO/IEC JTC 1/SC32 (WG1 & 2). The purpose of this liaison is to avoid possible duplication of efforts and the publication of conflicting specifications
+
+
+
+  - dates:
+      - 2004-09-10
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    identifier: 252
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved
+
+    actions:
+      - type: resolves
+        date_effective: 2004-09-10
+        message: ISO/TC154 becoming aware of the establishment of a JTC1 Web Services Study Group, is eager to ensure that  its suite of ebXML specifications that in themselves comprise a valuable form of web services is actively involve in the discussion. Therefore ISO/TC154 resolves to establish an active liaison with the JTC1 web services study group
+
+
+  - dates:
+      - 2004-09-10
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    identifier: 253
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved
+
+    actions:
+      - type: resolves
+        date_effective: 2004-09-10
+        message: |
+        ISO/TC154, noting that the chair and secretary duty of Mr. Vuilleumier ends in December 2004, resolves
+        - to thank Mr. Vuilleumier for agreeing to fulfill these duties at least for one more year (up to his foreseen retirement from the Swiss government, in December 2005)
+        - to identify which ISO/TC154 P-member could, in due time, take over these duties
+
+
+  - dates:
+      - 2004-09-10
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    identifier: 254
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved
+
+    actions:
+      - type: thanks
+        date_effective: 2004-09-10
+        message: ISO/TC154 thanks Mark Crawford and LMI, having provided very efficiently the hosting for this meeting
+
+
+  - dates:
+      - 2004-09-10
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Next ISO/TC154 meetings"
+    identifier: 255
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved
+
+    actions:
+      - type: resolves
+        date_effective: 2004-09-10
+        message: |
+        23rd: 2005-03 (possibly back to back before the UN/CEFACT Forum)
+        24th: 2005-09 (possibly back to back before the UN/CEFACT Forum)
```

Comment:
Here, I'm using `resolves` action type but not sure about it.

Source:
![capture13](https://user-images.githubusercontent.com/33627611/71759960-68fe0b00-2e8c-11ea-9ac3-655b07b56993.JPG)